runtime/cere-dev/src/lib.rs

```diff
@@ -342,6 +342,21 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+parameter_types! {
+	pub const PreimageMaxSize: u32 = 4096 * 1024;
+	pub const PreimageBaseDeposit: Balance = deposit(2, 64);
+	pub const PreimageByteDeposit: Balance = deposit(0, 1);
```

Comment:
If my calculation a correct it's quite a small amount. Will it be enough not to clog the state?

Reply:
This is how it's set to Polkadot
https://github.com/paritytech/polkadot/blob/c22e1c4173bf6966f5d1980f4299f7abe836f0c1/runtime/polkadot/src/lib.rs#L225

Reply:
DOT is more expensive than Cere so it might work for them but not for us. 
In the future, let's go through all the configurations and estimate attack costs